website/docs/r/organization.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: organization"
+sidebar_current: "docs-mongodbatlas-resource-organization"
+description: |-
+    Provides a Organization resource.
+---
+
+# Resource: mongodbatlas_organization
+
+`mongodbatlas_organization` provides a Organization resource. This allows organization to be created.
+
+~> **IMPORTANT NOTE:**  Creating an organization with this resource creates initial public and private key values that will need to be used in a separate terraform module/blueprint to permit access to new organization.
```

Comment:
Not sure what this means, can you rephrase/make more clear? If easier feel free to borrow language from Atlas Admin API docs - https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Organizations/operation/createOrganization

Reply:
Still could use improvement but not blocking on it.

Reply:
thanks. updated directly incase you want to take another look @themantissa